Lib/calendar.py

```diff
@@ -274,8 +305,8 @@ def yeardays2calendar(self, year, width=3):
         zero.
         """
         months = [
```

Comment:
monthdays2calendar one line (Part 1)
```suggestion
        months = [self.monthdays2calendar(year, m) for m in Month]
```